buku

```diff
@@ -2186,12 +2189,13 @@ class BukuDb:
                 LOGERR('No matching index %s', index)
                 return False
 
+            single_record = len(results) == 1
             if self.json is None:
                 print_rec_with_filter(results, self.field_filter)
             elif self.json:
-                write_string_to_file(format_json(results, True, self.field_filter), self.json)
+                write_string_to_file(format_json(results, single_record, field_filter=self.field_filter), self.json)
             else:
-                print_json_safe(results, True, self.field_filter)
+                print_json_safe(results, single_record, field_filter=self.field_filter)
```

Comment:
…Technically this block can be removed along with `return True` (and the `single_record` thing moved to the near-identical piece of code below); but I wasn't sure if you'd accept changing output format when a single-value range is supplied :sweat_smile: